tests/contrib/wsgi/test_wsgi.py

```diff
@@ -30,11 +30,20 @@ def chunked_response_generator_error(start_response):
             raise GeneratorExit()
 
 
+def raise_stop_iteration_exception(start_response):
+    status = "200 OK"
+    headers = [("Content-type", "text/plain")]
+    start_response(status, headers)
+    next(iter([]))
```

Comment:
is this the actual use case?

I would think it is `return iter([])`, no?

Reply:
Updated with a comment